server/accounts.go

```diff
@@ -949,6 +1011,17 @@ func (s *Server) updateAccountClaims(a *Account, ac *jwt.AccountClaims) {
 		c.applyAccountLimits()
 		c.mu.Unlock()
 	}
+
+	// Check if the signing keys changed, might have to evict
+	if signersChanged {
+		for _, c := range clients {
+			if c.user.SigningKey != "" {
```

Comment:
Wonder if it is safe to access `c.user.SigningKey` here without lock.

Reply:
Right now we only assign when the client is created - when we have the server update a connection by re-issuing a request to re-authenticate, we'll need to put this under a lock.